test/Organizer/OrganizerTest.php

```diff
@@ -285,6 +286,40 @@ function (Organizer $organizer) use ($initialAddress, $updatedAddress, $language
             );
     }
 
+    /**
+     * @test
+     */
+    public function it_can_set_an_initial_address_and_remove_it_later()
+    {
+        $initialAddress = new Address(
+            new Street('Wetstraat 1'),
+            new PostalCode('1000'),
+            new Locality('Brussel'),
+            Country::fromNative('BE')
+        );
+
+        $language = $this->organizerCreatedWithUniqueWebsite->getMainLanguage();
+
+        $this->scenario
+            ->given([$this->organizerCreatedWithUniqueWebsite])
+            ->when(
+                function (Organizer $organizer) use ($initialAddress, $language) {
+                    $organizer->updateAddress($initialAddress, $language);
+
+                    // Remove the address twice with the same value so we can
+                    // test it doesn't get recorded the second time.
+                    $organizer->removeAddress();
+                    $organizer->removeAddress();
+                }
+            )
+            ->then(
+                [
+                    new AddressUpdated($this->id, $initialAddress),
```

Comment:
Continuation from my previous comment:
Basically you could move this to `given` and remove `$organizer->updateAddress`